pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -183,7 +183,7 @@ func (this *SnapshotManager) CreateSnapshot(peID astrolabe.ProtectedEntityID, ta
 func (this *SnapshotManager) DeleteSnapshot(peID astrolabe.ProtectedEntityID) error {
 	log := this.WithField("peID", peID.String())
 	log.Infof("Step 1: Deleting the local snapshot")
-	err := this.DeleteProtectedEntitySnapshot(peID, false)
+	err := this.DeleteLocalProtectedEntitySnapshot(peID)
```

Comment:
Let's make the "Deleted the local snapshot" message an "else" so we don't get "Failed" then "Deleted"